From ff98aacb86a34215ab003a7952d18110d46c65c5 Mon Sep 17 00:00:00 2001
From: Andrei Botila <andrei.botila@nxp.com>
Date: Fri, 4 Feb 2022 11:35:21 +0200
Subject: [PATCH 17/43] Revert "s32-gen1: Enable ATF boot flow by default"

This reverts commit 1b0767fa69785108bc866684945720b682e0cdd1.
The S32_ATF_BOOT_FLOW has been removed from code. Since the removal of
STANDALONE support the S32_ATF_BOOT_FLOW is no longer necessary.

Issue: ALB-8347
Upstream-Status: Pending 

Signed-off-by: Andrei Botila <andrei.botila@nxp.com>
Signed-off-by: Zhantao Tang <zhantao.tang@windriver.com>
---
 configs/s32g274a_emu_defconfig          | 1 -
 configs/s32g274abluebox3_defconfig      | 1 -
 configs/s32g274abluebox3_qspi_defconfig | 1 -
 configs/s32g274ardb2_defconfig          | 1 -
 configs/s32g274ardb2_qspi_defconfig     | 1 -
 configs/s32g2xxaevb_defconfig           | 1 -
 configs/s32g2xxaevb_qspi_defconfig      | 1 -
 configs/s32g399a_emu_defconfig          | 1 -
 configs/s32g3xxaevb_defconfig           | 1 -
 configs/s32g3xxaevb_qspi_defconfig      | 1 -
 configs/s32r45_emu_defconfig            | 1 -
 configs/s32r45evb_defconfig             | 1 -
 configs/s32r45evb_qspi_defconfig        | 1 -
 13 files changed, 13 deletions(-)

diff --git a/configs/s32g274a_emu_defconfig b/configs/s32g274a_emu_defconfig
index 9aa28cad13..dbf0065762 100644
--- a/configs/s32g274a_emu_defconfig
+++ b/configs/s32g274a_emu_defconfig
@@ -4,7 +4,6 @@ CONFIG_ENV_SIZE=0x2000
 CONFIG_ENV_OFFSET=0x1e0000
 CONFIG_NR_DRAM_BANKS=3
 CONFIG_TARGET_S32G274AEMU=y
-CONFIG_S32_ATF_BOOT_FLOW=y
 CONFIG_DISTRO_DEFAULTS=y
 # CONFIG_SYS_MALLOC_CLEAR_ON_INIT is not set
 CONFIG_TOOLS_DEBUG=y
diff --git a/configs/s32g274abluebox3_defconfig b/configs/s32g274abluebox3_defconfig
index 32ee727bb2..b6717def8d 100644
--- a/configs/s32g274abluebox3_defconfig
+++ b/configs/s32g274abluebox3_defconfig
@@ -5,7 +5,6 @@ CONFIG_ENV_OFFSET=0x1e0000
 CONFIG_NR_DRAM_BANKS=3
 CONFIG_TARGET_S32G274ABLUEBOX3=y
 # CONFIG_S32GEN1_DRAM_INLINE_ECC is not set
-CONFIG_S32_ATF_BOOT_FLOW=y
 CONFIG_DISTRO_DEFAULTS=y
 CONFIG_TOOLS_DEBUG=y
 CONFIG_FIT=y
diff --git a/configs/s32g274abluebox3_qspi_defconfig b/configs/s32g274abluebox3_qspi_defconfig
index 3e4f670c09..a5edfc7598 100644
--- a/configs/s32g274abluebox3_qspi_defconfig
+++ b/configs/s32g274abluebox3_qspi_defconfig
@@ -6,7 +6,6 @@ CONFIG_ENV_OFFSET=0x1e0000
 CONFIG_NR_DRAM_BANKS=3
 CONFIG_TARGET_S32G274ABLUEBOX3=y
 # CONFIG_S32GEN1_DRAM_INLINE_ECC is not set
-CONFIG_S32_ATF_BOOT_FLOW=y
 CONFIG_DISTRO_DEFAULTS=y
 CONFIG_TOOLS_DEBUG=y
 CONFIG_FIT=y
diff --git a/configs/s32g274ardb2_defconfig b/configs/s32g274ardb2_defconfig
index 920317cdeb..6d8fbc1b01 100644
--- a/configs/s32g274ardb2_defconfig
+++ b/configs/s32g274ardb2_defconfig
@@ -3,7 +3,6 @@ CONFIG_ARCH_S32=y
 CONFIG_ENV_SIZE=0x2000
 CONFIG_ENV_OFFSET=0x1e0000
 CONFIG_NR_DRAM_BANKS=3
-CONFIG_S32_ATF_BOOT_FLOW=y
 CONFIG_DISTRO_DEFAULTS=y
 CONFIG_TOOLS_DEBUG=y
 CONFIG_FIT=y
diff --git a/configs/s32g274ardb2_qspi_defconfig b/configs/s32g274ardb2_qspi_defconfig
index b8d095c619..2dc8bd4a50 100644
--- a/configs/s32g274ardb2_qspi_defconfig
+++ b/configs/s32g274ardb2_qspi_defconfig
@@ -4,7 +4,6 @@ CONFIG_ENV_SIZE=0x10000
 CONFIG_ENV_SECT_SIZE=0x10000
 CONFIG_ENV_OFFSET=0x1e0000
 CONFIG_NR_DRAM_BANKS=3
-CONFIG_S32_ATF_BOOT_FLOW=y
 CONFIG_DISTRO_DEFAULTS=y
 CONFIG_TOOLS_DEBUG=y
 CONFIG_FIT=y
diff --git a/configs/s32g2xxaevb_defconfig b/configs/s32g2xxaevb_defconfig
index 31f181786d..fe856309d0 100644
--- a/configs/s32g2xxaevb_defconfig
+++ b/configs/s32g2xxaevb_defconfig
@@ -4,7 +4,6 @@ CONFIG_ENV_SIZE=0x2000
 CONFIG_ENV_OFFSET=0x1e0000
 CONFIG_NR_DRAM_BANKS=3
 CONFIG_TARGET_S32G2XXAEVB=y
-CONFIG_S32_ATF_BOOT_FLOW=y
 CONFIG_DISTRO_DEFAULTS=y
 CONFIG_TOOLS_DEBUG=y
 CONFIG_FIT=y
diff --git a/configs/s32g2xxaevb_qspi_defconfig b/configs/s32g2xxaevb_qspi_defconfig
index e2cc3deab0..9ee798a8c0 100644
--- a/configs/s32g2xxaevb_qspi_defconfig
+++ b/configs/s32g2xxaevb_qspi_defconfig
@@ -5,7 +5,6 @@ CONFIG_ENV_SECT_SIZE=0x10000
 CONFIG_ENV_OFFSET=0x1e0000
 CONFIG_NR_DRAM_BANKS=3
 CONFIG_TARGET_S32G2XXAEVB=y
-CONFIG_S32_ATF_BOOT_FLOW=y
 CONFIG_DISTRO_DEFAULTS=y
 CONFIG_TOOLS_DEBUG=y
 CONFIG_FIT=y
diff --git a/configs/s32g399a_emu_defconfig b/configs/s32g399a_emu_defconfig
index ec1aeb2e5c..8c64361f72 100644
--- a/configs/s32g399a_emu_defconfig
+++ b/configs/s32g399a_emu_defconfig
@@ -4,7 +4,6 @@ CONFIG_ENV_SIZE=0x2000
 CONFIG_ENV_OFFSET=0x1e0000
 CONFIG_NR_DRAM_BANKS=3
 CONFIG_TARGET_S32G399AEMU=y
-CONFIG_S32_ATF_BOOT_FLOW=y
 CONFIG_DISTRO_DEFAULTS=y
 # CONFIG_SYS_MALLOC_CLEAR_ON_INIT is not set
 CONFIG_TOOLS_DEBUG=y
diff --git a/configs/s32g3xxaevb_defconfig b/configs/s32g3xxaevb_defconfig
index 1e6e71b4d3..5a4915fae3 100644
--- a/configs/s32g3xxaevb_defconfig
+++ b/configs/s32g3xxaevb_defconfig
@@ -4,7 +4,6 @@ CONFIG_ENV_SIZE=0x2000
 CONFIG_ENV_OFFSET=0x1e0000
 CONFIG_NR_DRAM_BANKS=3
 CONFIG_TARGET_S32G3XXAEVB=y
-CONFIG_S32_ATF_BOOT_FLOW=y
 CONFIG_DISTRO_DEFAULTS=y
 CONFIG_TOOLS_DEBUG=y
 CONFIG_FIT=y
diff --git a/configs/s32g3xxaevb_qspi_defconfig b/configs/s32g3xxaevb_qspi_defconfig
index 9d36bef689..c0a224a8fe 100644
--- a/configs/s32g3xxaevb_qspi_defconfig
+++ b/configs/s32g3xxaevb_qspi_defconfig
@@ -5,7 +5,6 @@ CONFIG_ENV_SECT_SIZE=0x10000
 CONFIG_ENV_OFFSET=0x1e0000
 CONFIG_NR_DRAM_BANKS=3
 CONFIG_TARGET_S32G3XXAEVB=y
-CONFIG_S32_ATF_BOOT_FLOW=y
 CONFIG_DISTRO_DEFAULTS=y
 CONFIG_TOOLS_DEBUG=y
 CONFIG_FIT=y
diff --git a/configs/s32r45_emu_defconfig b/configs/s32r45_emu_defconfig
index b6f1461026..c7477498d6 100644
--- a/configs/s32r45_emu_defconfig
+++ b/configs/s32r45_emu_defconfig
@@ -3,7 +3,6 @@ CONFIG_ARCH_S32=y
 CONFIG_ENV_SIZE=0x2000
 CONFIG_NR_DRAM_BANKS=3
 CONFIG_TARGET_S32R45EMU=y
-CONFIG_S32_ATF_BOOT_FLOW=y
 CONFIG_DISTRO_DEFAULTS=y
 # CONFIG_SYS_MALLOC_CLEAR_ON_INIT is not set
 CONFIG_TOOLS_DEBUG=y
diff --git a/configs/s32r45evb_defconfig b/configs/s32r45evb_defconfig
index 7c4b34501b..fcd0786dbb 100644
--- a/configs/s32r45evb_defconfig
+++ b/configs/s32r45evb_defconfig
@@ -4,7 +4,6 @@ CONFIG_ENV_SIZE=0x2000
 CONFIG_ENV_OFFSET=0x1e0000
 CONFIG_NR_DRAM_BANKS=3
 CONFIG_TARGET_S32R45EVB=y
-CONFIG_S32_ATF_BOOT_FLOW=y
 CONFIG_DISTRO_DEFAULTS=y
 # CONFIG_SYS_MALLOC_CLEAR_ON_INIT is not set
 CONFIG_TOOLS_DEBUG=y
diff --git a/configs/s32r45evb_qspi_defconfig b/configs/s32r45evb_qspi_defconfig
index 051845ee38..cdbf9aec34 100644
--- a/configs/s32r45evb_qspi_defconfig
+++ b/configs/s32r45evb_qspi_defconfig
@@ -5,7 +5,6 @@ CONFIG_ENV_SECT_SIZE=0x10000
 CONFIG_ENV_OFFSET=0x1e0000
 CONFIG_NR_DRAM_BANKS=3
 CONFIG_TARGET_S32R45EVB=y
-CONFIG_S32_ATF_BOOT_FLOW=y
 CONFIG_DISTRO_DEFAULTS=y
 # CONFIG_SYS_MALLOC_CLEAR_ON_INIT is not set
 CONFIG_TOOLS_DEBUG=y
-- 
2.17.1

